sdk/lib/rtl/actctx.c

```diff
@@ -5149,6 +5184,12 @@ void actctx_init(PVOID* pOldShimData)
     {
         DPRINT1("Failed to create the implicit act ctx. Status: 0x%x!!!\n", Status);
     }
+
+#ifdef __REACTOS__
+    NtCurrentTeb()->ProcessEnvironmentBlock->ActivationContextData = process_actctx->ActivationContextData;
+#else
```

Comment:
Could ProcessEnvironmentBlock/Peb be a union? Not in this PR but in general.

Reply:
No it's not, because this would mean, having two different names for the same thing.
https://www.vergiliusproject.com/kernels/x86/Windows%202003/SP2/_TEB
https://www.vergiliusproject.com/kernels/x64/Windows%2011/23H2%20(2023%20Update)/_TEB
Like ours, it's called `ProcessEnvironmentBlock`.

Reply:
Well yes, one correct name and one for Wine.

Reply:
This would pollute our headers with wine ~~garbage~~

Maybe by including [wine/winternl.h](https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/wine/winternl.h;h=d011e65927e62436cc81fc33d83eeb00e71403bc;hb=HEAD#l366) only in this actctx.c file may do the trick (only if it doesn't include also our more correct headers).